searchcore/src/vespa/searchcore/config/partitions.def

```diff
@@ -50,7 +50,7 @@ dataset[].mpp			int default=1
 dataset[].maxnodesdownperfixedrow int default=0
 
 ## Use simple roundrobin or random.
```

Comment:
Consider updating comment to reflect new name (weighted)

Reply:
Agree - it's good if we can change everywhere.